src/components/CreateCourseRunPage/index.jsx

```diff
@@ -138,7 +138,8 @@ class CreateCourseRunPage extends React.Component {
     const parsedTypeOptions = courseOptionsData && courseOptionsData.type &&
       parseCourseTypeOptions(courseOptionsData.type.type_options);
     const courseRunTypeOptions = parsedTypeOptions && parsedTypeOptions.courseRunTypeOptions;
-    const defaultRunType = courseRunTypeOptions && courseRunTypeOptions[type][1].value;
+    const defaultRunType = courseRunTypeOptions && courseRunTypeOptions[type] &&
+      courseRunTypeOptions[type][1].value;
 
```

Comment:
defaultRunType will be `undefined` if there are not atleast 2 elements in the `courseRunTypeOptions[type]` array.  is this expected?

Reply:
Yeah, no problems with that. Just sets the initialValues array and it being undefined there will just leave it as forcing them to select an option.

Reply:
great